CONTRIBUTING.md

```diff
@@ -130,5 +130,5 @@ This project is implemented using Go and uses the standard golang tools for deve
 
 For setting up a Development environment on your local host, see the detailed instructions [here](./BUILD.md).
 
-The NDM design document is available [here](./docs/design.md).
+The Upgrade design document is available [here](./docs/design.md).
```

Comment:
`./docs/design.md` This file does not exist in the repo. 
May need to change to `upgrade.md` or `migration.md`